examples/step-66/doc/kind

```diff
@@ -0,0 +1 @@
+unfinished
```

Comment:
I guess we can move this to the `techniques` category as the tutorial is essentially finished?